chia/cmds/wallet_funcs.py

```diff
@@ -314,11 +315,11 @@ async def send(
                 ).to_tx_config(mojo_per_unit, config, fingerprint),
                 final_fee,
                 memos,
-                puzzle_decorator_override=[
-                    {"decorator": PuzzleDecoratorType.CLAWBACK.name, "clawback_timelock": clawback_time_lock}
-                ]
-                if clawback_time_lock > 0
-                else None,
```

Comment:
I guess pre-commit is passing but it's confusing that this change needs to be made.

Reply:
I see it was made due to a newer version of black.  For future references, try to keep this stuff out of the diff but this is small enough that I can resolve this :+1: